From 435152ebb74dcbd4a3a3316eec548d52ef034387 Mon Sep 17 00:00:00 2001
From: Benoit Bouchaud <benoit.bouchaud@raccourci.fr>
Date: Tue, 20 Oct 2020 17:04:09 +0200
Subject: [PATCH] remove PLL_ACF

---
 integrations/acf/load.php | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/integrations/acf/load.php b/integrations/acf/load.php
index 5be8998..09acdd0 100644
--- a/integrations/acf/load.php
+++ b/integrations/acf/load.php
@@ -7,11 +7,11 @@ if ( ! defined( 'ABSPATH' ) ) {
 	exit; // Don't access directly.
 };
 
-add_action(
-	'after_setup_theme',
-	function() {
-		if ( defined( 'ACF_VERSION' ) && version_compare( ACF_VERSION, '5.7.11', '>=' ) ) {
-			add_action( 'init', array( PLL_Integrations::instance()->acf = new PLL_ACF(), 'init' ) );
-		}
-	}
-);
+// add_action(
+// 	'after_setup_theme',
+// 	function() {
+// 		if ( defined( 'ACF_VERSION' ) && version_compare( ACF_VERSION, '5.7.11', '>=' ) ) {
+// 			add_action( 'init', array( PLL_Integrations::instance()->acf = new PLL_ACF(), 'init' ) );
+// 		}
+// 	}
+// );
-- 
2.27.0

